From c194b05d6b1fb98da896784c580ecba5c36b1297 Mon Sep 17 00:00:00 2001
From: Marius Brehler <marius.brehler@amd.com>
Date: Tue, 25 Mar 2025 12:13:13 +0000
Subject: [PATCH 11/17] Fix finding LAPACK and CBLAS

Use target names provided by `find_package()` instead of assuming that
the reference NETLIB libraries should be used.
---
 projects/hipblas/clients/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/projects/hipblas/clients/CMakeLists.txt b/projects/hipblas/clients/CMakeLists.txt
index a230135a10..b5e4eb0fb6 100644
--- a/projects/hipblas/clients/CMakeLists.txt
+++ b/projects/hipblas/clients/CMakeLists.txt
@@ -28,8 +28,8 @@ function( get_lapack lapack_lib lapack_inc )
     set( inc "${BUILD_DIR}/deps/deps-install/include" )
     set( ${cblas_inc} ${inc} PARENT_SCOPE )
   else()
-    find_package( lapack REQUIRED CONFIG )
-    set( lib "lapack" )
+    find_package( LAPACK REQUIRED )
+    set( lib "${LAPACK_LIBRARIES}" )
   endif()
   set( ${lapack_lib} ${lib} PARENT_SCOPE )
 endfunction( )
@@ -52,7 +52,7 @@ function( get_cblas cblas_libs cblas_inc )
       set( ${cblas_inc} ${inc} PARENT_SCOPE )
     else()
       find_package( cblas REQUIRED CONFIG )
-      set( libs cblas blas )
+      set( libs ${CBLAS_LIBRARIES} )
     endif()
   endif()
   set( ${cblas_libs} ${libs} PARENT_SCOPE )
@@ -221,7 +221,7 @@ if( BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS)
   find_package( GTest REQUIRED )
 
   message(STATUS "Build Dir: ${BUILD_DIR}")
-  message(STATUS "Linking Ref. Libs: ${BLAS_LIBRARY}")
+  message(STATUS "Linking Libs: ${BLAS_LIBRARY}")
 
   if( BUILD_CLIENTS_TESTS )
     add_subdirectory( gtest )
-- 
2.43.0

